thinkhazard/static/less/index.less

```diff
@@ -47,3 +47,8 @@
 .hazard-types-list li .hazard-icon {
   .box-shadow(0px 0px 20px -1px rgba(0, 0, 0, 0.15));
 }
+
+.index-brand {
```

Comment:
You don't need to create a new class here.
The `index.less` file is supposed to be compiled in `index.css` which is loaded only in `index.jinja2`.
It means that you should be able to override the font-size using `.logo-brand`.